site/docs/tools/export_import.md

```diff
@@ -51,6 +94,11 @@ want to overwrite an existing Nessie repository, then use the `--erase-before-im
     java -jar nessie-quarkus-cli-x.y.z-runner.jar help import
     ```
 
+## Migrating from a legacy version store type
+
+The CLI tool can be used to fully migrate a Nessie repository from one version store type to 
+another, see [Migration](../tools/migration.md) for a detailed example.
+
 ## Building blocks
```

Comment:
From this line one, I find the contents too developer-oriented to be of any interest for users trying to use Quarkus CLI. Alse, there is mention of key-lists and commit optimization that I think are not relevant anymore. I would be in favor of removing everything from this line til the end, but wanted to know first what others think.